src/lighteval/metrics/utils/extractive_match_utils.py

```diff
@@ -39,17 +39,33 @@
 from lighteval.utils.timeout import timeout
 
 
+@requires_latex2sympy2_extended
+def latex_normalization_config_default_factory():
+    from latex2sympy2_extended.latex2sympy2 import NormalizationConfig
+
+    return NormalizationConfig(
+        basic_latex=True,
+        units=True,
+        malformed_operators=True,
+        nits=True,
+        boxed=True,
+        equations=True,
+    )
+
+
 @dataclass(frozen=True)
 class LatexExtractionConfig:
     """Config for extracting latex from the prediction.
 
     Attributes:
         try_extract_without_anchor (bool): Whether to try extracting latex without requiring specific anchors like "answer:" or "final answer is"
-        enforce_boxed_match (bool): Whether to also consider extracting from plain \boxed{...} expressions
+        boxed_match_priority (int): Priority of the boxed match regex (-1 never, 0 first, 55 after final answer: anchor, etc...)
```

Comment:
How did you choose -1/0/55? are they standard/do they come from somewhere?

Reply:
The priorities are defined in lazy_regex_xxxx, each regex has priority when it's tried. There is not exact method how I chose the priorities, but they are jumping by 50/100 so that there is enough space for some adjustments.